bench/app.py

```diff
@@ -100,7 +105,7 @@ def remove_from_excluded_apps_txt(app, bench_path='.'):
 
 def get_app(git_url, branch=None, bench_path='.', skip_assets=False, verbose=False, postprocess=True, overwrite=False):
 	if not os.path.exists(git_url):
-		if not check_url(git_url, raise_err=False):
```

Comment:
Not sure if this is already on your to-do, but we should remove the `check_url` function as well

Reply:
Not at the moment; There are a few dead functions floating around bench. We'll need a cleanup, we could even remove all the frappe < v5 support :P

Reply:
Oh, for sure